README.md

```diff
@@ -87,20 +87,18 @@ The Azure Pipelines Infracost tasks can be used with either Azure Repos (only gi
 
           # Run Infracost and generate the JSON output, the following docs might be useful:
           # Multi-project/workspaces: https://www.infracost.io/docs/features/config_file
-          # Combine Infracost JSON files: https://www.infracost.io/docs/features/cli_commands/#combined-output-formats
           - bash: infracost breakdown --path=$(TF_ROOT)/plan.json --format=json --out-file=/tmp/infracost.json
             displayName: Run Infracost
 
-          # See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
-          - task: InfracostComment@0
+          # Add a cost estimate comment to a Azure Repos pull request.
+          - bash: |
+              infracost comment azure-repos \
+                 --path /tmp/infracost.json \
+                 --azure-repos-token $(System.AccessToken) \
```

Comment:
My bad, missed that too:
```suggestion
                 --azure-access-token $(System.AccessToken) \
```

Reply:
👍 , fixed in https://github.com/infracost/infracost-azure-devops/pull/45/commits/4fa1b8f25a406aab41ae47519cc8f587351957d1